nncf/torch/quantization/init_range.py

```diff
@@ -250,8 +253,11 @@ def __init__(
         self.batch_size = batch_size
 
     def _get_fwd_hook(self, collector: TensorStatisticCollectorBase) -> Callable:
+        from nncf.torch import no_nncf_trace
+
         def fwd_hook(module, input_, output):
-            collector.register_input(input_[0])
+            with no_nncf_trace():
+                collector.register_input(PTNNCFTensor(input_[0]))
```

Comment:
For Torch the tensor wrapping is also done at the hooks collecting the function/module inputs.